src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -2,16 +2,16 @@
 
 import org.junit.Assume;
 import org.junit.rules.MethodRule;
+import org.junit.runners.model.Annotatable;
 import org.junit.runners.model.FrameworkMethod;
 import org.junit.runners.model.Statement;
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
+import java.lang.annotation.*;
```

Comment:
\[Checkstyle\] WARNING: Using the '\.\*' form of import should be avoided \- java\.lang\.annotation\.\*\.